File: BASIC/1038.py
# author - newguo@sonaspy.cn
# coding - utf_8
n = eval(input())
l1 = list(map(int, input().split()))
l2 = list(map(int, input().split()))
mp = dict()
for i in range(101):
    mp[i] = 0
for i in l1:
    if i in mp.keys():
        mp[i] += 1
for i in range(1, l2[0]+1):
    if i == l2[0]:
        print(mp[l2[i]], end='')
    else:
        print(mp[l2[i]], end=' ')

File: BASIC/1038.cpp
// author - sonaspy@outlook.com
// coding - utf_8

#include <iostream>
#include <unordered_map>
#define test() freopen("in", "r", stdin)

using namespace std;

int main(int argc, char const *argv[])
{
    /* code */
    //test();
    unordered_map<int, int> mp;
    int n, score, flag = 1;
    cin >> n;
    for (int i = 0; i < n; i++)
        {
            scanf("%d", &score);
            mp[score] += 1;
        }
    cin >> n;
    for (int i = 0; i < n; i++)
        {
            scanf("%d", &score);
            if (flag)
                {
                    printf("%d", mp[score]);
                    flag = 0;
                }
            else
                printf(" %d", mp[score]);
        }
    return 0;
}